app/services/analytics_events.rb

```diff
@@ -2311,5 +2311,35 @@ def user_registration_email_confirmation(
       **extra,
     )
   end
+
+  # Tracks exceptions that are raised when running GetUspsProofingResultsJob
+  # @param [String] reason why was the exception raised?
+  # @param [String] enrollment_id
+  # @param [String] exception_class
+  # @param [String] exception_message
+  def idv_in_person_usps_proofing_results_job_exception(
+    reason:, enrollment_id:, exception_class: nil, exception_message: nil, **extra
+  )
+    track_event(
+      'GetUspsProofingResultsJob: Exception raised',
+      reason: reason,
+      enrollment_id: enrollment_id,
+      exception_class: exception_class,
+      exception_message: exception_message,
+      **extra,
+    )
+  end
+
+  # Tracks individual enrollments that fail during GetUspsProofingResultsJob
+  # @param [String] reason why did this enrollment fail?
+  # @param [String] enrollment_id
+  def idv_in_person_usps_proofing_results_job_enrollment_failure(reason:, enrollment_id:, **extra)
```

Comment:
should we also add a success event too? or an "updated" event with a success boolean? That way we can check the status of something just using logs, without having to dig in the database

Reply:
I created LG-6964 to add a success event.